adagrams/game.py

```diff
@@ -1,11 +1,141 @@
+import random
+from random import choices
+import copy
+
+LETTER_DICT = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+
+    letter_bank = []
+    letter_copy = copy.deepcopy(LETTER_DICT)
+
+    while len(letter_bank) < 10:
+        key_list = list(letter_copy.keys())
+        random_letter = random.choice(key_list)
+        if letter_copy[random_letter] > 0:
+            letter_bank.append(random_letter)
+            letter_copy[random_letter] -= 1
+    print(letter_bank)
+    return letter_bank 
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+
+    case_word = word.upper()
+    letter_bank_copy = copy.deepcopy(letter_bank)
+    for letter in case_word:
```

Comment:
You could simplify this conditonal if you added a `.upper()` on case_word. It looks like you already used `.upper()` when you create the variable but you also used `.upper()`  on the letter as well. Something like this:
```
for letter in word.upper():
        if letter in letter_bank_copy:
            letter_bank_copy.remove(letter)
        else:
            return False

    return True
```

Reply:
You could also just used an else instead of elif with a condition!